docs/using/event-types.md

```diff
@@ -57,6 +57,12 @@ curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-t
   "category": "business",
   "partition_strategy": "random",
   "enrichment_strategies": ["metadata_enrichment"],
+  "default_statistic": {
+    "messages_per_minute": 1000,	
+    "message_size":	5,
+    "read_parallelism":	10,
+    "write_parallelism": 5
```

Comment:
@v-stepanov good point. We should probably stick to 1 partition by default. Should I change `read_parallelism` to `1`? I'll have to update the Nakadi readme as well.